src/Nethermind/Nethermind.Merge.Plugin.Test/EngineModuleTests.V1.cs

```diff
@@ -1517,6 +1517,18 @@ public async Task payloadV1_latest_block_after_reorg()
         }
     }
 
+    [Test]
+    public async Task Should_return_ClientVersionV1()
+    {
+        using MergeTestBlockchain chain = await CreateBlockchain();
+        IEngineRpcModule rpcModule = CreateEngineModule(chain);
+        ResultWrapper<ClientVersionV1[]> result = rpcModule.engine_getClientVersionV1(new ClientVersionV1());
+        result.Data[0].Code.Should().Be(ProductInfo.ClientCode);
+        result.Data[0].Version.Should().Be(ProductInfo.Version);
+        result.Data[0].Name.Should().Be(ProductInfo.Name);
+        result.Data[0].Commit.Should().Be(ProductInfo.Commit);
```

Comment:
Minor: Would use collection/whole object assert rather than doing it property by property (in AAA you should have only 1 assertion)